include/identity_key.h

```diff
@@ -0,0 +1,121 @@
+/*
+ * Copyright (c) 2022 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef IDENTITY_KEY_H_
+#define IDENTITY_KEY_H_
+
+#include <stdint.h>
+
+/**
+ * @file
+ * @defgroup identity_key Identity key APIs
+ * @{
+ *
+ * @brief API for identity key on CryptoCell devices with KMU
+ */
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/** @brief Identity key size in bytes, corresponding to ECC secp256r1 */
+#define IDENTITY_KEY_SIZE_BYTES                 (32)
+
+/** @brief Error value when MKEK is missing from the KMU */
+#define ERR_IDENTITY_KEY_MKEK_MISSING           (0x15501)
+
+/** @brief Error value when identity key is missing from the KMU */
+#define ERR_IDENTITY_KEY_MISSING                (0x15502)
+
+/** @brief Error value when identity key can't be read */
+#define ERR_IDENTITY_KEY_READ_FAILED            (0x15503)
+
+/** @brief Error value when identity key can't be generated */
+#define ERR_IDENTITY_KEY_GENERATE_FAILED        (0x15504)
+
+/** @brief Error value when the identity key can't be exported as a public key */
```

Comment:
```suggestion
/** @brief Error value when the public part of the identity key can't be exported */
```

The old phrasing made it seem like we were exporting the whole key.

Reply:
Fixed